operators/src/source/gdal_source.rs

```diff
@@ -820,771 +691,281 @@ mod tests {
-        let dataset_y_pixel_size = -0.1;
-        let dataset_geo_transform = GeoTransform::new(
-            dataset_upper_right_coord,
-            dataset_x_pixel_size,
-            dataset_y_pixel_size,
-        );
-
-        let time_interval_provider = TimeIntervalInformation {
-            start_time: TimeInstance::from_millis(0),
-            time_step: TimeStep {
-                granularity: TimeGranularity::Seconds,
-                step: 1,
-            },
-        };
-
-        let gdal_params = GdalSourceParameters {
-            dataset_id: "modis_ndvi".to_owned(),
-            channel: None,
-        };
-
-        let dataset_information = JsonDatasetInformation {
-            time: time_interval_provider,
-            tile: None,
-            file_name_with_time_placeholder: "MOD13A2_M_NDVI_2014-01-01.TIFF".into(),
-            time_format: "".into(),
-            base_path: "../modis_ndvi".into(),
-            data_type: RasterDataType::U8,
-            geo_transform: dataset_geo_transform,
-            grid_shape: global_size_in_pixels.into(),
-            spatial_ref: SpatialReference::epsg_4326().into(),
-            measurement: Measurement::Unitless,
-        };
-
-        let dataset_information_provider = JsonDatasetInformationProvider {
-            dataset_information,
-            raster_data_root: "../operators/test-data/raster".into(),
-        };
-
-        let gdal_source = GdalSourceProcessor::<_, u8> {
-            dataset_information: dataset_information_provider,
-            gdal_params,
-            tiling_specification: TilingSpecification {
-                origin_coordinate: Coordinate2D::new(0., 0.),
-                tile_size_in_pixels: GridShape2D::from(tile_size_in_pixels),
-            },
-            phantom_data: PhantomData,
-        };
-
-        let tile_geo_transform = GeoTransform::new(
-            Coordinate2D::new(0., 0.),
-            dataset_x_pixel_size,
-            dataset_y_pixel_size,
-        );
+    fn test_load_tile_data() {
+        let output_shape: GridShape2D = [8, 8].into();
+        let output_bounds = BoundingBox2D::new_unchecked((-180., -90.).into(), (180., 90.).into());
 
-        // let x_tiles = 3600 / 600; // 6 => min tile position = -3
-        // let y_tiles = 1800 / 600; // 3 -> a split at 0,0 forces 4 tiles on the y-axis. => min tile position = -2
+        let x = load_ndvi_jan_2014(output_shape, output_bounds).unwrap();
 
-        let tile_information = TileInformation::new(
-            [-2, -3].into(),
-            tile_size_in_pixels.into(),
-            tile_geo_transform,
-        );
-        let time_interval = TimeInterval::new_unchecked(0, 1000);
+        assert_eq!(x.data.len(), 64);
 
-        let x = gdal_source
-            .load_tile_data(time_interval, tile_information)
-            .unwrap();
-
-        assert_eq!(x.tile_information(), tile_information);
-        assert_eq!(x.time, time_interval);
+        // TODO: compare actual data
```

Comment:
add the test

Reply:
I am not sure how to verfy the results. They are slightly differnent in QGIS but looks good to me 🤓

Reply:
well then we need a task to generate valid tests

Reply:
https://trello.com/c/nH4Rw9zK

Reply:
you should validate the looks good values. it is better then no test.

Reply:
2fb88fa6eae13d7e3c6a90eedc19c8ff5dcc5d35